### PR TITLE
fix(publish): use file name as label when multiple

### PR DESCRIPTION
**Pull request by nfriend**
### What does this PR do?

Fixes https://github.com/semantic-release/gitlab/issues/158 by using the correct property name (`label`) that is used as the link name in the GitLab release.

### Screenshots

Given a configuration like this:

```json
{
  "plugins": [
    [
      "@semantic-release/gitlab",
      {
        "assets": [
          { "path": "src/**/*", "label": "Source files" },
          { "path": "build/**/*", "label": "Compiled files" }
        ]
      }
    ]
  ]
}

```

**Before:**

When a glob pattern matched multiple files (just `build/**/*` in this example), each matched file was uploaded with the provided name ("Compiled files"), causing a validation error for every file but the first. 

before

**After:**

When a glob pattern matches multiple files, each matched file is uploaded with its file name instead of the provided `label`.

after

---

Closes https://github.com/semantic-release/gitlab/issues/158

### CODE REVIEW

**Comment by nfriend** [comment]
@gr2m @pvdlg @travi Would one of you mind reviewing this PR and merging if it looks good to you (when you have time)? 🙏 

Apologies for the ping, I'm not sure what the normal process is to request a review 🤔 

Thanks!

**Review by travi — APPROVED** [review]

**Comment by travi** [comment]
i'm not a gitlab user either, so i'm not very familiar with the needs of gitlab either. i agree that it is worth merging to unblock this issue.

@nfriend since none of the current maintainers use gitlab, it is admittedly a bit more difficult to support this plugin effectively on our own. would you be interested in helping with this plugin?

**Comment by nfriend** [comment]
> would you be interested in helping with this plugin?

@travi I'd be happy to! Let me know how I can help out.

**Comment by gr2m** [comment]
Thanks for offering to help Nathan! First thing would be to watch the repository. Maybe triage the open issues and pull requests to see if they are still relevant, if you find some time.

We want to formalize the process of onboarding contributors/maintainers, but there is some chore work we need to finish first

**Comment by nfriend** [comment]
@gr2m Sounds good! I've done an initial pass through the open issues and will continue to watch the activity going forward. I'll also try and push to resolve some of the bugs on GitLab's side (in particular, [<span>#</span>223832](https://gitlab.com/gitlab-org/gitlab/-/issues/223832#note_460224688)).

**Comment by gr2m** [comment]
awesome, thanks a lot!

**Pull request merged**
